sale_operating_unit/tests/test_sale_operating_unit.py

```diff
@@ -2,56 +2,58 @@
 
-class TestSaleOperatingUnit(common.TransactionCase):
+from odoo.addons.operating_unit.tests.common import OperatingUnitCommon
+
+
+class TestSaleOperatingUnit(OperatingUnitCommon):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls.res_groups = cls.env["res.groups"]
-        cls.partner_model = cls.env["res.partner"]
-        cls.res_users_model = cls.env["res.users"]
         cls.sale_model = cls.env["sale.order"]
         cls.sale_line_model = cls.env["sale.order.line"]
         cls.sale_team_model = cls.env["crm.team"]
         cls.acc_move_model = cls.env["account.move"]
-        cls.res_company_model = cls.env["res.company"]
         cls.product_model = cls.env["product.product"]
-        cls.operating_unit_model = cls.env["operating.unit"]
-        cls.company_model = cls.env["res.company"]
         cls.payment_model = cls.env["sale.advance.payment.inv"]
         # Company
-        cls.company = cls.env.ref("base.main_company")
         cls.grp_sale_user = cls.env.ref("sales_team.group_sale_manager")
         cls.grp_acc_user = cls.env.ref("account.group_account_invoice")
-        # Main Operating Unit
-        cls.ou1 = cls.env.ref("operating_unit.main_operating_unit")
-        # B2B Operating Unit
-        cls.b2b = cls.env.ref("operating_unit.b2b_operating_unit")
-        # B2C Operating Unit
-        cls.b2c = cls.env.ref("operating_unit.b2c_operating_unit")
         # Payment Term
         cls.pay = cls.env.ref("account.account_payment_term_immediate")
         # Customer
         cls.customer = cls.env.ref("base.res_partner_2")
         # Price list
         cls.pricelist = cls.env["product.pricelist"].search([], limit=1)
-        # Partner
-        cls.partner1 = cls.env.ref("base.res_partner_1")
         # Products
         cls.product1 = cls.env.ref("product.product_product_2")
         cls.product1.write({"invoice_policy": "order"})
-        # Create user1
-        cls.user1 = cls._create_user(
-            "user_1",
-            [cls.grp_sale_user, cls.grp_acc_user],
-            cls.company,
-            [cls.ou1, cls.b2c],
+        # Update users
+        cls.user1.write(
+            {
+                "groups_id": [
+                    Command.link(cls.grp_sale_user.id),
+                    Command.link(cls.grp_acc_user.id),
+                ],
+                "operating_unit_ids": [
+                    Command.link(cls.ou1.id),
+                    Command.link(cls.b2c.id),
+                ],
+            }
         )
-        # Create user2
-        cls.user2 = cls._create_user(
-            "user_2", [cls.grp_sale_user, cls.grp_acc_user], cls.company, [cls.b2c]
+        cls.user2.write(
+            {
+                "groups_id": [
+                    Command.link(cls.grp_sale_user.id),
+                    Command.link(cls.grp_acc_user.id),
+                ],
+                "operating_unit_ids": [
+                    Command.set(cls.b2c.id),
```

Comment:
`set` isn't the same as `link`
so I think you might be able to link like this `[Command.link(ou.id) for ou in operating_units],` if there are multiple units or just `Command.link(cls.b2c.id)`